C7/Map/TntLayer.cs

```diff
@@ -0,0 +1,42 @@
+using C7GameData;
+using Godot;
+using Resource = C7GameData.Resource;
+using Serilog;
+
+namespace C7.Map
+{
+	/// <summary>
+	/// Displays terrain yield overlays (from the tnt.pcx file).  These are most well known for letting you know where
+	/// there are bonus grasslands.
+	/// Note: I don't know why it's called tnt.
+	/// </summary>
+	public class TntLayer : LooseLayer
+	{
+		private ILogger log = LogManager.ForContext<ResourceLayer>();
```

Comment:
Are these layers sharing a log context or is this supposed to be TntLayer? A more generic copy-friendly but slightly inefficient way to do this might be something like ForContext<this.GetType())>() (I have not tested that)

Reply:
It looks like it isn't possible to use this.GetType() because this is technically in the initializer.  But that would be great, because it's definitely supposed to be TntLayer, it's a copy-paste error.  It's just complex enough syntax that I tend to copy and modify it, and sometimes forget the modify part.